.changeset/rich-insects-scream.md

```diff
@@ -0,0 +1,44 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Add configurable XML namespaces support to sitemap generation
+
+- Add `namespaces` option to control which XML namespaces are included
+- Support for news, xhtml, image, and video namespaces
+- All namespaces enabled by default for backward compatibility
+
+Example: Removing the video namespace
+If you want to exclude the video namespace from your sitemap, set video: false in your configuration:
+
+```
+// astro.config.mjs
+import { sitemap } from '@astrojs/sitemap';
+
+export default {
+  integrations: [
+    sitemap({
+      namespaces: {
+        video: false,
+        // other namespaces remain enabled by default
+      }
+    })
+  ]
+};
+```
+
+The generated XML will not include the xmlns:video namespace:
```

Comment:
```suggestion
The generated XML will not include the `xmlns:video` namespace:
```
I'm guessing this is the kind of thing we'd inline code for formatting. I LOVE that you provided the XML code block below! (See my other comment re: whether you think making a diff makes sense)